### PR TITLE
Update find and avoid example according to deepbots 0.1.3-dev2

### PR DESCRIPTION
**Pull request by eakirtas**
Fix bug that are related to the deepbots updates

### CODE REVIEW

File: examples/find_and_avoid/controllers/supervisor/supervisor.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+import gym
 import numpy as np
-
-import utilities as utils
 from deepbots.supervisor.controllers.supervisor_emitter_receiver import \
     SupervisorCSV
 from deepbots.supervisor.wrappers.keyboard_printer import KeyboardPrinter
-from deepbots.supervisor.wrappers.tensorboard_wrapper import TensorboardLogger
+from deepbots.supervisor.wrappers.tensorboard_wrapper import \
+    TorchTensorboardLogger
```
Comment (KelvinYang0320) [inline examples/find_and_avoid/controllers/supervisor/supervisor.py]: Is this <code>TensorboardLogger</code>?
I cannot run the code with <code>TorchTensorboardLogger</code>.
[Link to deepbots/supervisor/wrappers/tensorboard_wrapper](https://github.com/aidudezzz/deepbots/blob/851cba96c71ea13736dbe161ec90c230b193356d/deepbots/supervisor/wrappers/tensorboard_wrapper.py#L7)

File: examples/find_and_avoid/worlds/small_world.wbt [hunk 2]
```diff
@@ -1,35 +1,18 @@
...
 WoodenBox {
   translation -0.23 0.15 0.15
   size 0.1 0.3 0.7
 }
 DEF target Ball {
-  translation 0.38 0.06991244453845845 -0.37
+  translation 0.38 0.06991368534122255 -0.37
   radius 0.07
 }
 Viewpoint {
```
Comment (KelvinYang0320) [inline examples/find_and_avoid/worlds/small_world.wbt]: Maybe we can move viewpoint node to the 2nd position.
There are some warning message when the viewpoint node are not  included at position 2.

File: examples/find_and_avoid/controllers/supervisor/supervisor.py [hunk 3]
```diff
@@ -1,12 +1,14 @@
 import math
```
Comment (KelvinYang0320) [inline examples/find_and_avoid/controllers/supervisor/supervisor.py]: It seems that we import math for `math.pi`.
Maybe we can just use `np.pi`?

File: examples/find_and_avoid/controllers/supervisor/supervisor.py [hunk 4]
```diff
@@ -107,13 +104,13 @@ def get_reward(self, action):
...
                 self.should_done = True
             return -1
-        elif np.max(rf_values) > 100:
+        elif np.max(rf_values) > 200:
             return -0.5
 
-        if (distance != 0):
-            reward = (0.6 / distance)
+        # if (distance != 0):
+        #     reward = 0.1 * np.round((0.6 / distance), 1)
```
Comment (KelvinYang0320) [inline examples/find_and_avoid/controllers/supervisor/supervisor.py:106]: We can keep these comments for turning reward function.
#41

File: examples/find_and_avoid/controllers/supervisor/supervisor.py [hunk 5]
```diff
@@ -107,13 +104,13 @@ def get_reward(self, action):
...
 
-        reward -= (self.steps / self.steps_threshold)
+        # reward -= (self.steps / self.steps_threshold)
```
Comment (KelvinYang0320) [inline examples/find_and_avoid/controllers/supervisor/supervisor.py:108]: We can keep these comment for turning reward function.
#41

**Review by KelvinYang0320 — COMMENTED** [review]
I have left some comments. :grinning:

**Comment by eakirtas** [comment]
Thank you @KelvinYang0320. I will go through comments as soon as possible

File: examples/find_and_avoid/controllers/supervisor/supervisor.py (hunk 1 above)
Comment (eakirtas) [reply examples/find_and_avoid/controllers/supervisor/supervisor.py]: Tensorboard logger has to be updated accordingly. So I remove this for now

File: examples/find_and_avoid/worlds/small_world.wbt (hunk 2 above)
Comment (eakirtas) [reply examples/find_and_avoid/worlds/small_world.wbt]: Could you please elaborate this. I am not sure what's the problem

Comment (KelvinYang0320) [reply examples/find_and_avoid/worlds/small_world.wbt]: @ManosMagnus I teseted the code on webots R2021a.
The following is the warning message.
warningfor
As you can see, the viewpoint node is currently at the position 5.
pos
We can move `Viewpoint{}` to position 2 in this small_world.wbt, or cut and past this node on webots GUI.

Comment (eakirtas) [reply examples/find_and_avoid/worlds/small_world.wbt]: Fixed!

**Review by KelvinYang0320 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**